emission/analysis/intake/segmentation/trip_segmentation_methods/dwell_segmentation_dist_filter.py

```diff
@@ -138,180 +171,15 @@ def segment_into_trips(self, timeseries, time_query, filtered_points_df):
             t_loop.elapsed
         )
 
-
-        # Since we only end a trip when we start a new trip, this means that
-        # the last trip that was pushed is ignored. Consider the example of
-        # 2016-02-22 when I took kids to karate. We arrived shortly after 4pm,
-        # so during that remote push, a trip end was not detected. And we got
-        # back home shortly after 5pm, so the trip end was only detected on the
-        # phone at 6pm. At that time, the following points were pushed:
-        # ..., 2016-02-22T16:04:02, 2016-02-22T16:49:34, 2016-02-22T16:49:50,
-        # ..., 2016-02-22T16:57:04
-        # Then, on the server, while iterating through the points, we detected
-        # a trip end at 16:04, and a new trip start at 16:49. But we did not
-        # detect the trip end at 16:57, because we didn't start a new point.
-        # This has two issues:
-        # - we won't see this trip until the next trip start, which may be on the next day
-        # - we won't see this trip at all, because when we run the pipeline the
-        # next time, we will only look at points from that time onwards. These
-        # points have been marked as "processed", so they won't even be considered.
-
-        # There are multiple potential fixes:
-        # - we can mark only the completed trips as processed. This will solve (2) above, but not (1)
-        # - we can mark a trip end based on the fact that we only push data
-        # when a trip ends, so if we have data, it means that the trip has been
-        # detected as ended on the phone.
-        # This seems a bit fragile - what if we start pushing incomplete trip
-        # data for efficiency reasons? Therefore, we also check to see if there
-        # is a trip_end_detected in this timeframe after the last point. If so,
-        # then we end the trip at the last point that we have.
```

Comment:
Future fix: can we put this into an issue or a PR comment? I want to remember *why* we had to add `stopped_moving_after_last` You can then add a link to the description as a comment here instead of bloating up the codebase

Reply:
https://github.com/e-mission/e-mission-docs/issues/1117